modules/dnn/src/dnn_utils.cpp

```diff
@@ -102,11 +102,18 @@ void blobFromImagesWithParams(InputArrayOfArrays images_, OutputArray blob_, con
 
     int nch = images[0].channels();
     Scalar scalefactor = param.scalefactor;
+    Scalar mean = param.mean;
 
     if (param.ddepth == CV_8U)
     {
         CV_Assert(scalefactor == Scalar::all(1.0) && "Scaling is not supported for CV_8U blob depth");
-        CV_Assert(param.mean == Scalar() && "Mean subtraction is not supported for CV_8U blob depth");
+        CV_Assert(mean == Scalar() && "Mean subtraction is not supported for CV_8U blob depth");
```

Comment:
I propose to revert `param.mean` here. In case if assert user should see function parameter name, but not some internal variable,

Reply:
We used this for consistency with the `scalefactor` check. If `param.mean` should be used, we should also use `param.scalefactor`, making both checks more explicit. Or what do you think?

Reply:
you are right, both checks.